cli/CommandExecutor.cpp

```diff
@@ -59,21 +59,19 @@ void executeDescribeDatabase(
   const CatalogRelation *relation;
   if (arguments->size() == 0) {
     for (const CatalogRelation &rel : catalog_database) {
-      int column_width = static_cast<int>(rel.getName().length());
       max_column_width =
-          max_column_width < column_width ? column_width : max_column_width;
+          std::max(static_cast<int>(rel.getName().length()), max_column_width);
     }
   } else {
     const ParseString &table_name = arguments->front();
     const std::string &table_name_val = table_name.value();
     relation = catalog_database.getRelationByName(table_name_val);
 
     if (relation == nullptr) {
-     THROW_SQL_ERROR_AT(&(arguments->front())) << " Unrecognized relation "  <<table_name_val;
+      THROW_SQL_ERROR_AT(&(arguments->front())) << " Unrecognized relation " << table_name_val;
     }
-    int column_width = static_cast<int>(relation->getName().length());
-    max_column_width =
-        max_column_width < column_width ? column_width : max_column_width;
+    max_column_width = std::max(static_cast<int>(relation->getName().length()),
+                                    max_column_width);
```

Comment:
For your current line wrap fashion, we don't do indentation like this. Instead, align `max_column_width` to `static_cast`, as they are both arguments to `std::max`.